### ISSUE TITLE
 Full datasets for training and testing 

### ISSUE THREAD

**Issue by ilyes495** [phbradley/alphafold_finetune#1]
Hi, thank you so much for this great work! 
I know that the full dataset will be made available prior to the manuscript publication, is it possible to give us an estimate time of when it will be made available? if this will happen in months, can you then help us download the dataset on our own? 

Thanks!

**Comment by phbradley** [comment]
Hi there, thanks for your interest! This should be ready, at least in preliminary form, in the next week or two. If longer, I will put an interim file up on dropbox and post a link here.

**Comment by phbradley** [comment]
Hi there, sorry for the delays. We are having a terrible time uploading the files to zenodo-- neither the web interface nor the command line/python interface seem to be able to handle the file sizes, so we are momentarily stuck. In the meantime, I uploaded a preliminary dataset with the fine-tuned parameters and the training and validation datasets to dropbox:

https://www.dropbox.com/s/k4gay3hwyq3k0rb/datasets_alphafold_finetune_v1_2022-08-02.tgz?dl=0

**Comment by ilyes495** [comment]
Awsome! Thank you so much for your great efforts.

**Comment by phbradley** [comment]
Once you download the .tgz file, copy it into the alphafold_finetune/ folder and uncompress it, something like 

tar -xzvf datasets_alphafold_finetune_v1_2022-08-02.tgz

that should create a new folder called datasets_alphafold_finetune/ and hopefully the other examples will work. Let me know if you run into trouble!

**Comment by LivC93** [comment]
Any news on the full dataset? Since the weights are released, just the full test set would suffice.
